modules/st2flow-canvas/collapse-button.js

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Extreme Networks, Inc.
+//
+// Unauthorized copying of this file, via any medium is strictly
+// prohibited. Proprietary and confidential. See the LICENSE file
```

Comment:
We can't just copy these files with the old proprietary license.
We'll need to first change the license to Apache 2.0 and remove Extreme Networks EULA-related things.

I created dedicated issue for that https://github.com/StackStorm/st2flow/issues/374 to track that as part of the transition.

Reply:
Should we even use st2flow anymore?  This PR really makes that separated repo unnecessary because all of the relevant components from st2flow were literally copied into this repo (via this PR).

Is there a legal reason we need the other repo?

cc @armab @m4dcoder

Reply:
Sure, we can archive `st2flow` repo once the functionality gets integrated into `st2web` in full.
However we'll need to update the license from the Extreme proprietary to OSS Apache 2.0 in any way.

Reply:
I agree.  commented as such here:
https://github.com/StackStorm/st2flow/issues/374#issuecomment-638873860